### PR TITLE
elebotate directory Structure and perform specific task

### PR DESCRIPTION
**Pull request by sonukumar-a11**
Any additional information for the reviewer to start
NA

How should this be manually tested?
We need window OS in which Rust is installed. then execute this rust program

Which Trello task and sub-task does this change relate to?

Are there any changes pending?
No
Does any team have to be notified of changes in this feature?
Yes
Definition of Done:
Is there >80% unit test code coverage?
Yes
if not please mention the Code Coverage of code.
Does this PR add new dependencies? If so, please list out the same.
Will this feature require a new piece of infrastructure to be implemented?
Is there appropriate logging included?
Does the project compile ok?
Yes
Have Clippy violations been fixed?
Yes
 Have Code is properly formatted?
Yes

### CODE REVIEW

File: Assignments/assignment7/src/string_manipulation.rs [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+/// #Return
+///
+/// Returns the String to give the desired output....
+pub fn _desideratum_output(iterator1: &str, iterator2: &str, iterator3: &str) -> String
+{
+    let mut position = 0;
+    let mut i: usize = 0;
+    let mut array: Vec<char> = Vec::new();
+
+    while i < iterator2.len() {
```
Comment (aman09v) [inline Assignments/assignment7/src/string_manipulation.rs:18]: use meaning full names 
do not use 'i'

File: Assignments/assignment7/src/task/generating_subset.rs [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+///
+/// Returns a vector<Sting> having all substring input
+
+pub fn _generating_substring(input_iterable: String) -> Vec<String> {
+    if input_iterable.is_empty() {
+        return vec!["".to_string()];
+    }
+    let mut rest_str: Vec<String> = Vec::new();
+    let mut substring: &str;
+    for i in 0..input_iterable.len() {
```
Comment (aman09v) [inline Assignments/assignment7/src/task/generating_subset.rs:17]: do not use i, j

**Review by aman09v — CHANGES_REQUESTED** [review]
changes suggested

File: Assignments/assignment7/src/string_manipulation.rs [hunk 3]
```diff
@@ -0,0 +1,54 @@
+/// This function take input to as
```
Comment (aman09v) [inline Assignments/assignment7/src/string_manipulation.rs:1]: use meaningful comments

**Review by aman09v — CHANGES_REQUESTED** [review]
use good comments

**Comment by sonukumar-a11** [comment]
Nothing

**Comment by sonukumar-a11** [comment]
Nothing

**Pull request closed without merging**